mentors/src/data_pipeline/ibm_transcript_service.py

```diff
@@ -0,0 +1,32 @@
+import os
+from watson_developer_cloud import SpeechToTextV1
+
+
+"""
+API credentials for the IBM Watson service are given below
+Web login credentials for https://console.ng.bluemix.net/
+Credentials File: Ask Ben Nye for it
+Place the text file containing the credentials file in the root of the website version code
+Contact Madhusudhan Krishnamachari at madhusudhank@icloud.com to get assistance on how to use Watson. (IBM Documentation should help)
+"""
+username = os.environ["WATSON_USERNAME"]
```

Comment:
should we raise an exception here if either `username` or `password` unset?

I'm thinking of the case of a new install where user tries to run the process without setting up `watson_credentials.txt`. They should get an error message that gives them clear instructions like "You need a file at secrets/watson_credentials with username and password to generate transcripts..."

Probably the correct place to put that would be in the `Makefile` instead (since this script doesn't require that file, just env vars)